aidants_connect_web/tests/test_views/test_new_mandat.py

```diff
@@ -224,31 +224,34 @@ def test_updating_autorisation_for_for_same_aidant(self):
         last_journal_entry = Journal.objects.last()
         self.assertEqual(last_journal_entry.action, "create_autorisation")
 
-        # second session : updating the autorisation
-        autorisation_builder_2 = Connection.objects.create(
+        # second session : 'updating' the autorisation
+        mandat_builder_2 = Connection.objects.create(
             usager=self.test_usager, demarches=["papiers"], duree_keyword="LONG"
         )
 
         session = self.client.session
-        session["connection"] = autorisation_builder_2.id
+        session["connection"] = mandat_builder_2.id
         session.save()
         # trigger the autorisation creation/update
         self.client.post(
             "/creation_mandat/recapitulatif/",
             data={"personal_data": True, "brief": True, "otp_token": "223456"},
         )
 
-        self.assertEqual(Autorisation.objects.count(), 1)
-        updated_autorisation = Autorisation.objects.get(
+        self.assertEqual(Autorisation.objects.count(), 2)
+        all_usager_aidant_papiers_autorisations = Autorisation.objects.filter(
```

Comment:
Idem que plus haut, les autorisation sont entre les organisation et les usager : 
```suggestion
        all_usager_organisation_papiers_autorisations = Autorisation.objects.filter(
```